src/common/sass/widgets/_panel.scss

```diff
@@ -35,7 +35,7 @@
     color: $inverse_secondary_fg_color;
   
     .app-menu-icon {
-      -st-icon-style: symbolic;
+      //-st-icon-style: symbolic;
```

Comment:
Any reason to keep this line around?

Reply:
Yes, but it also shouldn't have been commented out either. Thanks!